common/search/src/test/scala/weco/api/search/elasticsearch/ElasticsearchServiceTest.scala

```diff
@@ -101,13 +275,46 @@ class ElasticsearchServiceTest
         val multiSearchResponseFuture =
           elasticsearchService.executeMultiSearchRequest(multiSearchRequest)
 
-        whenReady(multiSearchResponseFuture) { multiSearchResponseEither =>
-          val multiSearchResponse = multiSearchResponseEither.right.value
-          val queryResults =
-            multiSearchResponse.items.map(_.response.right.value)
-          val returnedThings = queryResults.flatMap(_.to[ExampleThing])
+        whenReady(multiSearchResponseFuture) {
+          case (errors, searchResponses) =>
+            errors should have size (0)
+
+            val returnedThings = searchResponses.flatMap(_.to[ExampleThing])
+            returnedThings.toSet shouldBe thingsToQueryFor.toSet
+        }
+      }
+    }
+
+    it("returns an appropriate error when the specified index does not exist") {
```

Comment:
this test description feels a bit off to me – you're testing that you get an error *and* that you get the results that work